examples/multiomics_mogonet/model.py

```diff
@@ -0,0 +1,98 @@
+from typing import List, Optional
+
+from torch.nn import CrossEntropyLoss
+from yacs.config import CfgNode
+
+from kale.embed.mogonet import MogonetGCN
+from kale.loaddata.multiomics_datasets import SparseMultiOmicsDataset
+from kale.pipeline.multiomics_trainer import ModalityTrainer
```

Comment:
To discuss in person name for `ModalityTrainer`

Reply:
Changed.